qiskit/circuit/library/pauli_evolution.py

```diff
@@ -152,6 +158,94 @@ def time(self, time: ParameterValueType) -> None:
         """
         self.params = [time]
 
+    def inverse(self, annotated: bool = False) -> Gate:
+        """Invert this instruction by negating the time.
+
+        Args:
+            annotated: If set to ``True`` the output inverse gate will be returned
+                as :class:`.AnnotatedOperation`.
+
+        Returns:
+            The inverse operation.
+        """
+        if annotated:
+            return AnnotatedOperation(self, InverseModifier())
```

Comment:
Can you think of a use-case where it would be useful to have an inverse-annotated PauliEvolutionGate, instead of the simpler version that negates time? 

So far we have allowed `gate.inverse(annotated=True)` to produce "simple" gates when possible, for instance an inverse of an `HGate` is also an `HGate`, regardless of `annotated`.

The same question/comment also apply to `power`, and also possibly to `control` (given how compact the representation is with your change).